static/js/settings.js

```diff
@@ -10,13 +12,43 @@ const jsonToIdFields = new Map([
 ]);
 const idToJsonFields = new Map(Array.from(jsonToIdFields, (a) => a.reverse()));
 
-// Splits a string of names/numbers into a list when it finds "," or "\n"
-const normalizeExtraIds = (extraIds) => {
-  if (!extraIds) {
-    return [];
+const displayUnsavedChanges = () => {
+  const errors = document.getElementById('errors');
+  errors.innerText = 'Attention! Unsaved changes';
+};
+
+const deleteExtraID = (extraID) => {
+  if (extraIDs.has(extraID)) {
+    extraIDs.delete(extraID);
+  }
+  displayUnsavedChanges();
+  renderExtraIDs();
```

Comment:
If you didn't delete any id you should not rerender / displayUnsavedChanges.

Also you don't have to use .has(extraId)  but you can check the return value from delete https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/delete